registry/templates/create_bucket

```diff
@@ -1,3 +1,4 @@
+{{ if exists "/deis/store/gateway" }}
```

Comment:
It would, but in this case, the assumption in this script is that we are using the deis-store, which we are not.